### PR TITLE
Us/14 merchant invoices index

### PR DESCRIPTION
**Pull request by sicknacco**
This PR contains the work for user story 14 as well as some minor refactors.
Model tests for top 5 total revenue were fixed with adding an `end` and updating some of the expected statements. There is still a failing test on this page though.

Merchant invoice controller was updated with an index action
Merchant invoice index view all set up with invoice ids as links that lead to their individual show pages.

### CODE REVIEW

File: spec/features/merchants/invoices/index_spec.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      expect(page).to have_content("Invoice ##{@invoice9.id}")
+      expect(page).to have_content("Invoice ##{@invoice10.id}")
+      expect(page).to have_content("Invoice ##{@invoice11.id}")
+      expect(page).to have_content("Invoice ##{@invoice12.id}")
+      expect(page).to have_content("Invoice ##{@invoice13.id}")
+    end
+    it 'each id links to the merchant invoice show page' do
+      visit merchant_invoices_path(@merchant2)
+      
+      click_link("#{@invoice7.id}")
```
Comment (thawley2) [inline spec/features/merchants/invoices/index_spec.rb:25]: Great job testing out multiple links

File: spec/models/merchant_spec.rb
Comment (thawley2) [inline spec/models/merchant_spec.rb:1]: Thanks for fixing this page! Totally did not remember to check after messing with the test data.

**Review by thawley2 — APPROVED** [review]
Looking clean!!!

**Review by abwmn — APPROVED** [review]
looking good guys!

**Pull request merged**